System/Process.hs

```diff
@@ -652,6 +654,24 @@ getPid (ProcessHandle mh _ _) = do
     _ -> return Nothing
 
 
+-- ----------------------------------------------------------------------------
+-- getCurrentPid
+
+-- | Returns the PID (process ID) of the current process. On POSIX systems,
+-- this calls 'getProcessID' from "System.Posix.Process" in the @unix@ package.
+-- On Windows, this calls 'getCurrentProcessId' from "System.Win32.Process" in
+-- the @Win32@ package.
+--
+-- @since TODO
```

Comment:
We'll call this 1.6.12.0. Can you update this, as well as add an entry to the changelog? Other than that it looks ready to go, thank you!

Reply:
Done!